src/overcooked_ai_py/mdp/layout_generator.py

```diff
@@ -166,11 +198,11 @@ def dig_space_with_fringe_expansion(self, grid, prop_empty=0.1):
                 if grid.is_valid_dig_location(location):
                     fringe.add(location)
 
-    def add_features(self, grid, prop_features=0):
+    def add_features(self, grid, prop_features=0, feature_types=None):
         """
         Places one round of basic features and then adds random features 
         until prop_features of valid locations are filled"""
-        feature_types = [POT, ONION_DISPENSER, DISH_DISPENSER, SERVING_LOC] # NOTE: currently disabled TOMATO_DISPENSER
+        if not feature_types: feature_types = DEFAULT_FEATURE_TYPES
```

Comment:
@bmielnicki can we stremline the code to be `feature_types=DEFAULT_FEATURE_TYPES` in the argument stage?

Reply:
Main reason to do it this way is to prevent using mutables as default parameters. See "Mutable Default Arguments" section in https://docs.python-guide.org/writing/gotchas/ . 
It probably would not break the code now, but it is easy to break it by accident.
My bad that it does not work as intended, more safe (intended) version would be:
`if not feature_types: feature_types = copy.copy(DEFAULT_FEATURE_TYPES)`
Other approach is to use tuple as its immutable. I will change feature_types default parameter to the tuple.